### ISSUE TITLE
Could you provide a Docker version of the packaging tool?

### ISSUE THREAD

**Issue by martinzu** [bryangerlach/rdgen#7]
Dear developer, thank you very much for creating such a convenient tool to package Rustdesk. However, after deploying it, I found that the web configuration page works normally, but I receive an HTTP server 500 error after submitting. Therefore, I was wondering if you could provide a Docker version of the packaging service? Users would only need to configure parameters and environment variables. This would help avoid many issues related to installation and deployment. I wonder if this could be implemented? Thank you once again!

**Comment by bryangerlach** [comment]
I can look into creating a docker image. In the meantime, you can set DEBUG to True at line 32 of settings.py and run the server again. You should then get debug info about the error.

**Comment by martinzu** [comment]
Hi, I've turned on debug mode in settings.py like this:

![image](https://github.com/user-attachments/assets/d1d51273-54d7-43e2-a1bd-f69babe8e540)

But I still get a browser error when trying to compile:

```
{
    "error": "Something went wrong"
}
```

The log error is:

```
root@HomeLab:~/rdgen# python manage.py runserver 0.0.0.0:8000
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
January 01, 2025 - 13:44:45
Django version 5.1.4, using settings 'rdgen.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.

[01/Jan/2025 13:44:55] "GET / HTTP/1.1" 200 15454
<Response [401]>
[01/Jan/2025 13:46:05] "POST /generator HTTP/1.1" 200 33
<Response [401]>
[01/Jan/2025 13:46:39] "POST /generator HTTP/1.1" 200 33
^C(rdgen)
```

**Comment by bryangerlach** [comment]
So you are getting response 401 which means you are unauthorized. Did you follow the instructions to set up your github account?

**Comment by martinzu** [comment]
github secrets (setup on your github account for your rdgen repo):
GENURL="example.com:8000"
*this is the domain and port that your are running rdgen on, needs to be accesible on the internet, depending on how you have this setup the port may not be needed


I would like to ask, is this example.com:8000 an address that can be accessed from the external network? It is currently a host on my intranet. It may be a problem here.

**Comment by bryangerlach** [comment]
Yes, it will need to be accessed from external network. Basically, github actions will need to communicate with that server over the internet.

**Comment by delko-it** [comment]
Hello, trying to create Dockerfile, its quite simple:
```
# Dockerfile
FROM python:3.13

# Mounts the application code to the image
COPY src opt
WORKDIR /opt/rdgen
# setup a python virtual environment called rdgen
RUN python -m venv rdgen
RUN chmod +x rdgen/bin/activate
# activate the python virtual environment
RUN /bin/sh -c rdgen/bin/activate
# install the python dependencies
RUN pip install --no-cache-dir -r requirements.txt
# setup the database
RUN python manage.py migrate
# expose port 8000
EXPOSE 8000
# runs the production server
ENTRYPOINT ["python", "manage.py"]
CMD ["runserver", "0.0.0.0:8000"]
```
Docker image spins up, but i'm getting this errors:
```
[04/Jan/2025 19:42:59] "GET / HTTP/1.1" 200 15454
[04/Jan/2025 19:43:44] "POST /generator HTTP/1.1" 200 33
```

**Comment by bryangerlach** [comment]
Both your GET and POST are showing a response code of 200, which means everything is OK. If you login to github and open your rdgen repository, do you see any Actions running in the Actions tab?

**Comment by delko-it** [comment]
Workflows aren’t being run on this forked repository

**Comment by delko-it** [comment]
> Both your GET and POST are showing a response code of 200, which means everything is OK. If you login to github and open your rdgen repository, do you see any Actions running in the Actions tab?

It runs ok, i didn't enabled workflows on github. It was disabled by default.
